torch/_export/serde/serialize.py

```diff
@@ -457,7 +457,12 @@ def export_nn_module_stack(val):
                 path, ty = val
 
                 assert isinstance(path, str)
-                normalized_ty = ty.__module__ + "." + ty.__qualname__
+
+                if isinstance(ty, str):
```

Comment:
in what case does this happen?

Reply:
When we deserialize an exported file back to python ExportedProgram, node.meta['nn_module_stack'] was not deserialized back into its original object (which is a `type(module)`), it's a simply str instead. This is understandable, since the `type(module)` object might not be available in the deserialize's python env. 

Before:
`('', <class 'sigmoid.inference.test_model_generator.MySimpleModel'>)`

After:
('', 'sigmoid.inference.test_model_generator.MySimpleModel')

When we serialized such exported program again, we need to handle the case where node.meta['nn_module_stack'] is a string. 

I will add these as a comment.

Reply:
If we take one step further, we should just always save string in node.meta['nn_module_stack'] . 

`type(module)` is not serializable / deserializable.